phalcon/http/response/cookies.zep

```diff
@@ -40,7 +40,7 @@ class Cookies implements CookiesInterface, InjectionAwareInterface
 
 	protected _useEncryption = true;
 
-	protected _cookies;
+	protected _cookies = [];
```

Comment:
@SidRoberts Actually this will negatively impact performance in ZendEngine 2.x (PHP 5.x). Could you please refactor this to initialize property at construct time?

Refs: https://github.com/phalcon/zephir/issues/1255